docs/build/index.md

```diff
@@ -52,23 +162,117 @@ The code is hosted on <a href="https://github.com/Lauryy06/Upsilon">GitHub</a>.
 git clone https://github.com/Lauryy06/Upsilon.git
 ```
 
+<br>
+
 ## Run Upsilon on your computer
 
 Once the SDK has been installed, just open your terminal (Msys2, Terminal.app, xterm…) and type the following commands:
 
+You have a lot of different choices to run Upsilon (depending on your model...).
+The version of the hardware of the calculator is printed at its back.
+
+<br>
+
+### Simulator binaries for your platform
+
 ```
 make PLATFORM=simulator clean
 make PLATFORM=simulator epsilon_run
 ```
 
-## Run Upsilon on your calculator
+<br>
+
+### Model n0100
+
+(note: you can change the `EPSILON_I18N=en` flag to `fr`, `nl`, `pt`, `it`, `de`, `es` or `hu`).
+
+```bash
+make MODEL=n0100 clean
+make MODEL=n0100 EPSILON_I18N=en OMEGA_USERNAME="{Your name, max 15 characters}" -j4
+```
+
+Now, run either:
+
+```bash
+make MODEL=n0100 epsilon_flash
+```
+to directly flash the calculator after pressing simultaneously `reset` and `6` buttons and plugging in.
+
+<br>
 
-You can also update your NumWorks calculator easily. Note that you'll need to press the Reset button and that all data on your calculator will be lost.
+or:
 
+```bash
+make MODEL=n0100 OMEGA_USERNAME="" binpack -j4
 ```
+to make binpack witch you can flash to the calculator from [Ti-planet's webDFU](https://ti-planet.github.io/webdfu_numworks/n0100/). Binpacks are a great way to share a custom build of Upsilon to friends.
+
+<br>
+
+### Model n0110
+
+
+```bash
 make clean
-make
+make OMEGA_USERNAME="{Your name, max 15 characters}" -j4
+```
+
+Now, run either:
+
+```bash
 make epsilon_flash
 ```
+to directly flash the calculator after pressing simultaneously `reset` and `6` buttons and plugging in.
+
+<br>
+
+or:
+
+```bash
+make OMEGA_USERNAME="" binpack -j4
+```
+to make binpack witch you can flash to the calculator from [Ti-planet's webDFU](https://ti-planet.github.io/webdfu_numworks/n0110/). Binpacks are a great way to share a custom build of Upsilon to friends.
+
+<br>
+### Web simulator
+
+First, install emsdk :
+
+```bash
+git clone https://github.com/emscripten-core/emsdk.git
+cd emsdk
+./emsdk install latest-fastcomp
+./emsdk activate latest-fastcomp
+source emsdk_env.sh
+```
+
+Then, compile Upsilon :
+
+```bash
+make clean
+make PLATFORM=simulator TARGET=web OMEGA_USERNAME="{Your name, max 15 characters}" -j4
+```
+
+The simulator is now in `output/release/simulator/web/simulator.zip`
+
+<br>
+
+###3DS Simulator
```

Comment:
The Web simulator and 3DS Simulator title aren't correctly displayed on GitHub

Reply:
That's an error. Will be corrected.

Reply:
Thank you.

Reply:
Corrected.